ansible/install-apps.yml

```diff
@@ -0,0 +1,29 @@
+---
+- name: Install OS-level apps
+  hosts: all
+  become: yes
+  vars:
+  - apt_repo_list_file: /etc/apt/sources.list.d/google-chrome.list
+  tasks:
+  - stat:
+      path: "{{ apt_repo_list_file }}"
+    register: google_apt_exists
+    ignore_errors: yes
+  - name: dir
```

Comment:
Please use human-readable explanations for task names so that others could actually understand what is the effect of this task on the target.